sonar-core/src/main/java/org/sonar/core/rule/RuleDao.java

```diff
@@ -30,7 +30,9 @@
  * @deprecated in 4.4 moved to org.sonar.server.rule.db.RuleDao.
  */
 @Deprecated
-public class RuleDao implements BatchComponent, ServerComponent {
+@BatchSide
+@ServerSide
+public class RuleDao {
```

Comment:
![INFO](https://raw.githubusercontent.com/henryju/image-hosting/master/info.png)Do not forget to remove this deprecated code someday.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)